<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedTask</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Inserir">
     <attribute name="title">
      <string extracomment="Inserir">Inserir</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_nameInput"/>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QComboBox" name="selection_bar">
        <item>
         <property name="text">
          <string>Selecionar tipo...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Médico</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paciente</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selection_bar2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Selecione a prioridade...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prioridade alta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prioridade média</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prioridade baixa</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="layout_esp_2" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="layout_esp">
         <item>
          <widget class="QLabel" name="lb_esp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Especialização</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_esp">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="layout_pat" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="layout_esp_10">
         <item>
          <widget class="QLabel" name="lb_pat">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Patologia</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_pat">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_insert">
        <property name="text">
         <string>Inserir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>                                  Pacientes em espera                                                                                          Médicos disponíveis</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tbl_paciente">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Prioridade</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Patologia</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tbl_medico">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Especialização</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bt_atender">
        <property name="text">
         <string>Atender Paciente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_disponibilizar">
        <property name="text">
         <string>Disponibilizar Médico</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_NameSort">
          <property name="text">
           <string>Ordenar por Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_GradeSort">
          <property name="text">
           <string>Ordenar por Prioridade</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>611</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gb_avg">
         <property name="title">
          <string>Média de pacientes atendidos por médico:</string>
         </property>
         <widget class="QLabel" name="lbl_avg">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>331</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_highest">
         <property name="title">
          <string>Médico com maior quantidade de pacientes: </string>
         </property>
         <widget class="QLabel" name="lbl_highest">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>10</y>
            <width>341</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_lowest">
         <property name="title">
          <string>Médico com menor quantidade de pacientes: </string>
         </property>
         <widget class="QLabel" name="lbl_lowest">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>341</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Histórico</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>301</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prioridade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Patologia</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>241</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Especialização</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboMed">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Selecione um médico...</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboPac">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Selecione um paciente...</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Médico</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Paciente</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btn_NameSort</tabstop>
  <tabstop>le_nameInput</tabstop>
  <tabstop>selection_bar</tabstop>
  <tabstop>tbl_medico</tabstop>
  <tabstop>btn_GradeSort</tabstop>
  <tabstop>btn_insert</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboMed</tabstop>
  <tabstop>comboPac</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>selection_bar2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
